.github/workflows/ci.yml

```
@@ -10,7 +10,22 @@ on:
   workflow_dispatch:
 
 jobs:
-  build_test_publish:
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: taiki-e/install-action@cargo-hack
```

This is fine, but just in case you would want to extend this, I found this syntax cleaner as it allows multiple tools

```suggestion
      - uses: taiki-e/install-action@v2
        with: { tool: cargo-hack }
```